styles/elements/table/_core.scss

```diff
@@ -0,0 +1,72 @@
+// Table
+//
+// Table styling to use throughout the app.
+//
+// Markup:
+// <table class="table">
+//   <thead>
+//     <tr><th>Header</th></tr>
+//   </thead>
+//
+//   <tbody>
+//     <tr><td>Cell</td></tr>
+//   </tbody>
+// </table>
+//
+// Styleguide Style Toolbox - Elements - Table
+
+// Table CSS
+// -----------------------------------------------------------------------------
+
+
+%table {
```

Comment:
So related to comment about whether should provide a mixin or not, we could do both and let devs decide.

So create something like 
`@mixin base-table { ... }`  

then do
```
%table {
  @include base-table();
}
```
So then the dev can `@extend %table;` or `@include base-table();` in their project as needed :)